code/game/objects/items/battering_ram.dm

```diff
@@ -0,0 +1,117 @@
+/*
+ * BATTERING RAM
+ *
+ * KNOCK-KNOCK, its Justice!
+ */
+/obj/item/batteringram
+	icon = 'icons/obj/weapons/fireaxe.dmi'
+	icon_state = "battering_ram0"
+	base_icon_state = "battering_ram"
+	lefthand_file = 'icons/mob/inhands/weapons/axes_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/axes_righthand.dmi'
+	name = "battering ram"
+	desc = "An extremely heavy and unwieldy hunk of metal used to break down airlocks. Why would anyone ram a sliding airlock?"
+
+	w_class = WEIGHT_CLASS_HUGE
+	item_flags = SLOWS_WHILE_IN_HAND | IMMUTABLE_SLOW
+	slot_flags = ITEM_SLOT_BACK
+	obj_flags = CONDUCTS_ELECTRICITY
+	resistance_flags = FIRE_PROOF
+	armor_type = /datum/armor/item_batteringram
+
+	demolition_mod = 3 // 9 hits to down a airlock
```

Comment:
9 hits to down an airlock for a weapon that is owned by one person, is not replacable if it was stolen

rough

Reply:
this isnt really a code review